### PR TITLE
Update 'SDL2' package to v2.30.2

### PR DESCRIPTION
**Pull request by drodin**
* I've followed [this guide](https://hunter.readthedocs.io/en/latest/creating-new/update.html)
  step by step carefully. **Yes**

---

### CODE REVIEW

File: cmake/projects/SDL2/hunter.cmake [hunk 1]
```diff
@@ -193,6 +193,50 @@ hunter_add_version(
...
+        "https://github.com/libsdl-org/SDL/releases/download/release-2.26.5/SDL2-2.26.5.tar.gz"
+        SHA1
+        ca5d89edc537fd819eddab1f1a86f61e45fcb68b
+)
+
+hunter_add_version(
+        PACKAGE_NAME
+        SDL2
+        VERSION
+        "2.28.5"
```
Comment (NeroBurner) [inline cmake/projects/SDL2/hunter.cmake:222]: No need to add all the versions. Just the newest is enough

Comment (drodin) [reply cmake/projects/SDL2/hunter.cmake:222]: Those are major versions only. They break some stuff from version to version, and it's easier to try different versions through config than have a git submodule. I have checked building with all intermediate versions.

Comment (NeroBurner) [reply cmake/projects/SDL2/hunter.cmake:222]: ah ok, then I'm merging. Thanks for the clarification

**Review by NeroBurner — APPROVED** [review]

**Comment by NeroBurner** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.25.6

**Pull request merged**